specification.json

```diff
@@ -24,7 +24,7 @@
         {
             "id": "Requirement 1.1.4",
             "machine_id": "requirement_1_1_4",
-            "content": "The API MUST provide a function for retrieving the `provider` implementation.",
+            "content": "The API MUST provide a function for retrieving the metadata field of the configured `provider`.",
```

Comment:
I think this is good - it answers the query I had, and I think will avoid any confusion, or abuse of the API.